### PR TITLE
Fixes clumsiness

### PR DESCRIPTION
**Pull request by ghost**
PLEASE, find any other use of ```disabilities &```, and FIX IT!

### CODE REVIEW

**Comment by JohnGinnane** [comment]
Just did some looking around and found why this needs to be done.

Disabilities are now defined as strings, not numbers: https://github.com/HippieStation/HippieStation/blob/dcd3e383ff1d58bfa9ab3c3f53c3f287da86e80b/code/__DEFINES/stat.dm#L11

This means bitwise operations don't work. Here's more instances where our code is trying to use bitwise operations:

https://github.com/HippieStation/HippieStation/blob/5ad1592b1296925291c1713040e80694929fcf18/hippiestation/code/game/objects/items/butterfly_knives.dm#L45

https://github.com/HippieStation/HippieStation/blob/b12ca539e42de0b44d6b2c61565343171869936c/hippiestation/code/modules/mob/living/carbon/human/species_types/meeseeks.dm#L41

https://github.com/HippieStation/HippieStation/blob/70921fc1dd3262f46d0ff20f4c010d9554f28479/hippiestation/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L79

https://github.com/HippieStation/HippieStation/blob/e45f44e3ed400ff045db586dd7191fb20e09209d/hippiestation/code/game/machinery/trapdoors.dm#L101

https://github.com/HippieStation/HippieStation/blob/691e7d8bb9825e74e2b043df50314eabf5071871/hippiestation/code/datums/martial/sleeping_carp.dm#L203 (this isn't broken, but I think we should use a proc instead)

https://github.com/HippieStation/HippieStation/blob/dcd3e383ff1d58bfa9ab3c3f53c3f287da86e80b/hippiestation/code/modules/reagents/chemistry/reagents/toxin_reagents.dm#L437

I think we should use the tg procs for add/removing/checking for disabilities:

https://github.com/HippieStation/HippieStation/blob/dcd3e383ff1d58bfa9ab3c3f53c3f287da86e80b/code/modules/mob/living/status_procs.dm#L143

@MoreRobustThanYou do you want to do this?

**Review by JohnGinnane — DISMISSED** [review]
Code is 👌

**Comment by ghost** [comment]
I'll go ahead and fix these

**Comment by JohnGinnane** [comment]
thanks famalam

**Comment by ghost** [comment]
stuff fixed

File: hippiestation/code/modules/mob/living/carbon/human/species_types/meeseeks.dm [hunk 1]
```diff
@@ -38,7 +38,7 @@
 	H.adjustCloneLoss(0.3)
 	H.adjustBrainLoss(0.8)
 	if(stage_ticks == MEESEEKS_TICKS_STAGE_ONE)
-		H.disabilities |= CLUMSY
+		H.add_disability(CLUMSY, IM_MISTER_MEESEEKS)
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/carbon/human/species_types/meeseeks.dm]: what's this for?

File: hippiestation/code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 2]
```diff
@@ -434,7 +434,7 @@
 	if(prob(1))
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
-			H.disabilities |= CLUMSY
+			H.add_disability(CLUMSY, CHEMISTRY_BULLSHIT)
```
Comment (JohnGinnane) [inline hippiestation/code/modules/reagents/chemistry/reagents/toxin_reagents.dm:437]: And this?

File: hippiestation/code/modules/mob/living/carbon/human/species_types/meeseeks.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/mob/living/carbon/human/species_types/meeseeks.dm]: I dunno, I guess meeseeks have the clusmy disability

File: hippiestation/code/modules/reagents/chemistry/reagents/toxin_reagents.dm (hunk 2 above)
Comment (ghost) [reply hippiestation/code/modules/reagents/chemistry/reagents/toxin_reagents.dm:437]: apparently some chem

File: hippiestation/code/modules/mob/living/carbon/human/species_types/meeseeks.dm (hunk 1 above)
Comment (JohnGinnane) [reply hippiestation/code/modules/mob/living/carbon/human/species_types/meeseeks.dm]: I mean the "IM_MISTER_MEESEEKS" part? The second argument to `add_disability()` is a list of disabilities that are applied to `H`

Comment (ghost) [reply hippiestation/code/modules/mob/living/carbon/human/species_types/meeseeks.dm]: Changed to GENETIC_MUTATION

**Review by JohnGinnane — APPROVED** [review]
🆒 and ✔️

**Comment by JohnMcLaime** [comment]
Borgs are fucking clumsy too lmao

**Pull request merged**